avocado/core/sysinfo.py

```diff
@@ -412,6 +412,20 @@ def __init__(self, basedir=None, log_packages=None, profiler=None):
             log.debug('File %s does not exist.', commands_file)
             self.commands = []
 
+        fail_commands_file = settings.get_value('sysinfo.collectibles',
+                                                'fail_commands',
```

Comment:
I suggest adding `_file` to this option to make it more intuitive for the users.

Reply:
@beraldoleal I am looking to send a v2 of this with all comments addressed, so for my clarity, you mean: `fail_commands_file = settings.get_value('sysinfo.collectibles', 'fail_commands', ...` ?